src/main/java/com/epam/prejap/teatrees/game/Grid.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.prejap.teatrees.game;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Provides Playfield an api for working with 2D array.
+ * @author dominik_kaminski
+ * @author przemyslaw_szewczyk
+ */
+record Grid(byte[][] grid, int rows, int cols) {
```

Comment:
What is the benefit of having this class as a record? does it simplify the code in any way? (Considering the number of lines in it, I guess not really.) Equals is overridden, constructors are explicit, the content of the grid is mutable and it's accessible publicly.

Reply:
Changed to regular class.